backend/dal/Repositories/PropertyService.cs

```diff
@@ -100,26 +100,27 @@ public PimsProperty Get(int id)
                 .Include(p => p.PropertyDataSourceTypeCodeNavigation)
                 .Include(p => p.PropertyClassificationTypeCodeNavigation)
                 .Include(p => p.PimsPropPropTenureTypes)
+                    .ThenInclude(t => t.PropertyTenureTypeCodeNavigation)
```

Comment:
missing navigation include. Was causing an exception if there was a tenure on the property